samples/DemoScenarios/Contacts/Schema.xml

```diff
@@ -1,13 +1,19 @@
- <entities >
-       <entity name="contact" displayname="Contact" etc="2" primaryidfield="contactid" primarynamefield="fullname" disableplugins="false">
-          <fields>
-             <field displayname="First Name" name="firstname" type="string" />
-             <field displayname="Last Name" name="lastname" type="string" />
-             <field displayname="Parent Contact" name="parentcontactid" type="entityreference" lookupType="contact" />
-             <field displayname="Contact" name="contactid" type="guid" primaryKey="true" />
-             <field displayname="Home Phone" name="telephone2" type="string" />
-             <field displayname="Mobile Phone" name="mobilephone" type="string" />
-          </fields>
-          <relationships />
-       </entity>
- </entities>
\ No newline at end of file
+<entities >
+	<entity name="contact" displayname="Contact" etc="2" primaryidfield="contactid" primarynamefield="fullname" disableplugins="false">
+		<fields>
+			<field displayname="First Name" name="firstname" type="string" />
+			<field displayname="Last Name" name="lastname" type="string" />
+			<field displayname="Parent Contact" name="parentcontactid" type="entityreference" lookupType="contact" />
```

Comment:
Was this change intended? Seems as though it's for a separate demo scenario.

Reply:
This was just left over changes as I used the contact scenario when first developing the Obfuscation feature. I'll set it back to its original state.